app/consumer.py

```diff
@@ -0,0 +1,65 @@
+import base64
+import logging
+
+from sdx.common.async_consumer import AsyncConsumer
+from structlog import wrap_logger
+
+from app import settings
+from app.decrypter import Decrypter
+from app.sdxftp import SDXFTP
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+'''
+The AsyncConsumer will automatically pick up the RABBIT queue from a file called settings with var RABBIT_QUEUE
+TODO this really needs to change
+'''
+QUEUE = settings.RABBIT_QUEUE
+
+
+class Consumer(AsyncConsumer):
+
+    def __init__(self, log=None):
+        super().__init__(log)
+        self._decrypter = Decrypter(settings.RAS_SEFT_PUBLIC_KEY,
+                                    settings.SDX_SEFT_PRIVATE_KEY,
+                                    settings.SDX_SEFT_PRIVATE_KEY_PASSWORD)
+
+        self._ftp = SDXFTP(logger, settings.FTP_HOST, settings.FTP_USER, settings.FTP_PASS, settings.FTP_PORT)
+
+    def on_message(self, unused_channel, basic_deliver, properties, body):
+        encrypted_jwt = body.decode("utf-8")
+        logger.debug("Message Received")
```

Comment:
Should be `info` - we're always going to want to log the receipt of a message (ideally with an identifier [like `tx_id` if such a thing exists])